testsuite/tests/tool-ocamlopt-save-ir/save_ir_after_scheduling.ml

```diff
@@ -0,0 +1,15 @@
+(* TEST
+ * native-compiler
+ ** setup-ocamlopt.byte-build-env
+ *** ocamlopt.byte
+   flags = "-save-ir-after scheduling -S"
+   ocamlopt_byte_exit_status = "0"
+ **** check-ocamlopt.byte-output
+ ***** script
+   script = "sh ${test_source_directory}/save_ir_after_scheduling.sh"
```

Comment:
I'm surprised we need a shell script to check the existence of the `.cmir` files, but I'm not very familiar with `ocamltest`.  I suggest asking @shindere if he could review these few parts of the diff.

Reply:
(The testsuite changes have been reviewed separately)